src/components/Books/Books.tsx

```
@@ -0,0 +1,15 @@
+import React from "react";
+import styles from "../../styles/Books.module.scss";
+import FilterContainer from "./Filter/FilterContainer";
+import MainContainer from "./Main/MainContainer";
+type BooksProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)